lib/WatcherProcess.php

```diff
@@ -345,13 +345,13 @@ private function countCpuCores() {
                 $cmd = "wmic cpu get NumberOfCores";
                 break;
             case "linux":
-                $cmd = "nproc";
+            case "darwin":
+                $cmd = "getconf _NPROCESSORS_ONLN";
                 break;
+            case "netbsd":
+            case "openbsd":
             case "freebsd":
-                $cmd = "sysctl -a | grep 'hw.ncpu' | cut -d ':' -f2";
-                break;
-            case "darwin":
-                $cmd = "sysctl -a | grep 'hw.ncpu:' | awk '{ print $2 }'";
+                $cmd = "sysctl hw.ncpu | cut -d ':' -f2";
```

Comment:
Does it still need the `cut` in that case?

Reply:
Why not just `getconf NPROCESSORS_ONLN`?

Reply:
see https://github.com/amphp/aerys/pull/207#issuecomment-343936901

Reply:
Yes, output is `hw.ncpu: 8`.

Reply:
@staabm ?

```
rugged$ getconf NPROCESSORS_ONLN
4
rugged$ uname -a
NetBSD rugged 8.99.1 NetBSD 8.99.1 (GENERIC) #7: Mon Jul  3 15:56:30 CEST 2017  root@chieftec:/public/netbsd-root/sys/arch/amd64/compile/GENERIC amd64
```

Reply:
If it works there, too, we can certainly use it for NetBSD as well. Want to provide a PR?

Reply:
`sysctl` works too and it's not big gain to change it again, it would be just more elegant to use `getconf` in the first place.